mesonbuild/mintro.py

```diff
@@ -64,8 +63,7 @@ def __init__(self,
 
 def get_meson_introspection_types(coredata: T.Optional[cdata.CoreData] = None,
                                   builddata: T.Optional[build.Build] = None,
-                                  backend: T.Optional[backends.Backend] = None,
-                                  sourcedir: T.Optional[str] = None) -> 'T.Mapping[str, IntroCommand]':
+                                  backend: T.Optional[backends.Backend] = None) -> 'T.Mapping[str, IntroCommand]':
```

Comment:
Removing this unused (since it was first introduced!) parameter should be done in its own commit.